### PR TITLE
JD5NW-42: error handling

### PR DESCRIPTION
**Pull request by tymekf**
create error handler servlet
log errors

### CODE REVIEW

**Comment by isacademyci** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ActiveHallFilter.java#L45](https://github.com/infoshareacademy/jjdd5-niewiem/blob/9a51ec56ce5a36d7393fed14e95c9382c73ec501/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java#L45): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [ActiveHallFilter.java#L49](https://github.com/infoshareacademy/jjdd5-niewiem/blob/9a51ec56ce5a36d7393fed14e95c9382c73ec501/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java#L49): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java [hunk 1]
```diff
@@ -80,7 +81,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     }
 
     private boolean isURIExcludedFromFilter(String reqUri) {
-        LOG.debug("Requested URI: {}", reqUri);
+        LOG.info("Requested URI: {}", reqUri);
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java]: moze lepiej zostawic tu debug? @BertPie ?

File: ClubManagerWeb/src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -14,20 +14,91 @@
...
-        <user>root</user>
-        <password>pass123</password>
-
-        <!--infoShare Academy-->
-        <!--<url>jdbc:mysql://localhost:6603/club?useLegacyDatetimeCode=false&amp;serverTimezone=Europe/Warsaw</url>-->
+        <!--<url>jdbc:mysql://db:3306/club?useLegacyDatetimeCode=false&amp;serverTimezone=Europe/Warsaw</url>-->
         <!--<user>root</user>-->
         <!--<password>pass123</password>-->
 
+        <!--infoShare Academy-->
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/webapp/WEB-INF/web.xml]: tu zostawiamy db:3306 inaczej appka nie bedzie dzialac po zdeployowaniu przez jenkinsa

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java (hunk 1 above)
Comment (BertPie) [reply ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java]: Tak, to miało być zmienione na info tylko na czas jak chcieliśmy w logach zobaczyć zapytania jakie były. Teraz powinno być wrócone do debug.

**Comment by tymekf** [comment]
racja, obie rzeczy były tymczasowe i zmienię je jak będę mógł

wt., 8 sty 2019, 18:35: BertPie <notifications@github.com> napisał(a):

**Comment by isacademyci** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ActiveHallFilter.java#L45](https://github.com/infoshareacademy/jjdd5-niewiem/blob/429d37f59f25fcfc0eb08fc89846e5f59ed7275c/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java#L45): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [ActiveHallFilter.java#L49](https://github.com/infoshareacademy/jjdd5-niewiem/blob/429d37f59f25fcfc0eb08fc89846e5f59ed7275c/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/halls/filters/ActiveHallFilter.java#L49): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**